drive/carts/tween.lua

```diff
@@ -491,6 +491,7 @@ function tween_machine:add_tween(instance)
         start_time = 0,
         duration = 0,
         elapsed = 0,
+        frame = 0,
```

Comment:
Might be a hack. There is a tween when loading images. Frame is needed because the protocol for getting an image takes in how many bytes to read and the offset. That's because you can't load a full image in a single frame without getting fps throttle, so the idea was, read like 1/8th of the image each frame of an animation. A new cart image could load in like 8 frames while you are doing a screen swipe animation, so it could make it look seamless especially if the image is cached.